packages/fx-core/src/common/deps-checker/internal/funcToolChecker.ts

```diff
@@ -295,7 +296,7 @@ export class FuncToolChecker implements DepsChecker {
       DepsCheckerEvent.funcInstallScriptCompleted,
       async () => {
         await runWithProgressIndicator(
-          async () => await this.doInstallPortableFunc(FuncVersion.v3),
+          async () => await this.doInstallPortableFunc(FuncVersion.v4),
```

Comment:
What if user has node 12 and only uncheck the `Node Checker` setting?
How about adding some node version check here as well?

Reply:
Those three things are one thing for users who want to use node 12.
Node 12 user will see the following ouput below when local debugging the first time after upgrade Teams Toolkit. So uses can know how to work on node 12 correctly.

`To continue local debugging with Node.js v12, please go to Visual Studio Code setting page for Teams Toolkit and uncheck Node.js and Azure Function Core Tools. And use your Azure Function Core Tools V3.`

Reply:
Then there still be a scenario that user only unchecks node checker. In such case we'd better give failure message as early as possible.

Reply:
Hold on this scenario to receive more feedback from internal user.